source/opt/ir_context.h

```diff
@@ -59,13 +61,14 @@ class IRContext {
   friend inline Analysis operator<<(Analysis a, int shift);
   friend inline Analysis& operator<<=(Analysis& a, int shift);
 
-  // Create an |IRContext| that contains an owned |Module|
+  // Creates an |IRContext| that contains an owned |Module|
   IRContext(spvtools::MessageConsumer c)
       : unique_id_(0),
         module_(new Module()),
         consumer_(std::move(c)),
         def_use_mgr_(nullptr),
-        valid_analyses_(kAnalysisNone) {
+        valid_analyses_(kAnalysisNone),
+        constant_mgr_(nullptr) {
```

Comment:
Why didn't you initialize type_mgr_ also?

Reply:
Missed it.  Done.